style.css

```diff
@@ -0,0 +1,136 @@
+
+.basic-section {
+    width: 100%;
+    height: 100vh;
+    background-image: url("img/cover.jpg");
+    background-position: center;
+    background-size: cover;
+    opacity: 0.9;
+}
+
+body{
+    font-family: 'Montserrat', sans-serif;
+    background-color: rgb(48, 49, 70); 
+    /*margin: 70px;*/
+    margin-top: 0;
+    color: white;
+    max-width: 1440px;
```

Comment:
you set max width to the body. 
And set background image to the section. 
Imagine if the screen is wider than 1440px. How will look the background?